control_panel_api/tests/test_views.py

```diff
@@ -783,3 +783,27 @@ def test_proxy(self, mock_request, mock_k8s_config):
             headers={'authorization': self.K8S_AUTH_TOKEN},
             verify=self.K8S_SSL_CERT_PATH,
         )
+
+
+class TestPagination(AuthenticatedClientMixin, APITestCase):
```

Comment:
This test case could potentially be moved in a `test_pagination.py` file.

Reply:
Good idea